src/main/java/org/jvnet/hudson/test/WarExploder.java

```diff
@@ -84,23 +90,34 @@ private static File explode() throws Exception {
             }
         }
 
-        // locate jenkins.war
-        File war;
-        URL winstone = WarExploder.class.getResource("/winstone.jar");
```

Comment:
A lot of [reindentation](https://github.com/jenkinsci/jenkins-test-harness/pull/100/files?w=1); would be clearer if you could use an `if`-`else if`-`else` series with the same indentation.